data/transport missions.txt

```diff
@@ -1975,3 +1975,40 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "The Broker"
+	description "Transport the broker's enforcers to <destination> by <day>."
+	blocked "You'll need more room to house the hired thugs. Return when you have <capacity>."
+	deadline
+	cargo "Living necessities" 5
+	passengers 15
+	minor
+	to offer
+		not "year" == 3013
```

Comment:
I don't like this, partially because it lacks a `random` condition or anything similar, and it seems like a pretty arbitrary restriction to me.